app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -280,8 +317,54 @@ public void onClick(View v) {
         });
         //Show outdoor map on start
         showOutdoorMap();
+        relativeButtonThread();
+
+    }
+
+    /**
+     * Positions the Locate Me button as well as the
+     * Directions button relative to the bottomsheet at all times
+     */
+    private void relativeButtonThread(){
+        //Thread
+        Thread thread = new Thread(){
+            @Override
+            public void run(){
+                while(true) {
+                    try {
+                        sleep(20);
```

Comment:
no. change the height at transitions between fragments. Not by waiting 20 seconds between refresh